### ISSUE TITLE
Nvidia digital vibrance and gamma 

### ISSUE THREAD

**Issue by KrisDee1981** [Maassoft/ColorControl#300]
Hello. Is it possible to add Nvidia digital vibrance control?
I have AW3423DWF OLED monitor and I'm trying to clamp the colors a little bit in SDR only, but not to the sRGB level. 
Basically i would like to have 2 presets:
1.HDR on + 50% digital vibrance
2.HDR off + 40% digital vibrance

**Comment by Maassoft** [comment]
Yes, this will be fairly easy to implement. Will be added to a next release.

**Comment by KrisDee1981** [comment]
Thank you so much for this update. I know I could use Novideo sRGB but I don't like how colors look with sRGB, P3 or Adobe clamps in SDR games.
42-43% digital vibrance gives me little bit of extra saturation in SDR, but skin tones still looks natural.
